packages/hotpotqa/src/aviary/envs/hotpotqa/env.py

```diff
@@ -441,11 +447,14 @@ def construct_lookup_list(self, keyword: str) -> str:
         if not self.state.page:
             return "Lookup failed. You have not specified a Wikipedia page yet."
 
-        if self.state.last_lookup != keyword:
+        if (
+            self.state.last_action != self.tools[1].info.name
+            or self.state.last_lookup != keyword
+        ):
             self.state.last_lookup = keyword
             self.state.lookup_results = [
-                s.strip() + "."
-                for s in self.state.page.split(". ")
+                s.strip()
+                for s in self.state.page.split("\n")
```

Comment:
Is there an example of how this works?

Reply:
Like the Milhouse example you told me about

Reply:
Also here: https://github.com/Future-House/aviary/pull/96